konduit-serving-cli/pom.xml

```diff
@@ -105,6 +105,12 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>ai.konduit.serving</groupId>
+            <artifactId>konduit-serving-common-tests</artifactId>
+            <version>0.1.0-SNAPSHOT</version>
```

Comment:
For all of these: It's better practice to use `${project.version}` instead of the hardcoded version string like `0.1.0-SNAPSHOT`